src/mqt/bench/benchmarks/bv.py

```diff
@@ -0,0 +1,73 @@
+"""Bernstein-Vazirani benchmark definition."""
+
+from __future__ import annotations
+
+from qiskit import QuantumCircuit
+
+
+def create_circuit(num_qubits: int, dynamic: bool = False, hidden_string: str | None = None) -> QuantumCircuit:
+    """Create a quantum circuit for the Bernstein-Vazirani algorithm.
+
+    Arguments:
+        num_qubits: Total number of qubits in the circuit (including the flag qubit).
+        hidden_string: The hidden bitstring to be found (default: alternating pattern of 1 and 0).
+        dynamic: Whether to use a dynamic layout (default: False).
```

Comment:
The order of the arguments does not correspond to the parameter order in the function definition.